src/panels/config/cloud/alexa/cloud-alexa.ts

```diff
@@ -170,17 +175,38 @@ class CloudAlexa extends LitElement {
                 secondary-line
                 @click=${this._showMoreInfo}
               >
-                ${entity.entity_id in this.hass.entities
-                  ? html`<button
-                      class="link"
-                      .entityId=${entity.entity_id}
-                      @click=${this._openAliasesSettings}
-                    >
-                      ${this.hass.localize(
-                        "ui.panel.config.cloud.alexa.manage_aliases"
-                      )}
-                    </button>`
-                  : ""}
+                ${aliases
+                  ? html`
+                      <button
+                        class="link"
+                        .entityId=${entity.entity_id}
+                        @click=${this._openAliasesSettings}
+                        aria-label=${this.hass.localize(
+                          "ui.panel.config.cloud.alexa.manage_aliases"
+                        )}
+                      >
+                        ${aliases.length > 0
+                          ? [...aliases]
+                              .sort((a, b) =>
+                                stringCompare(a, b, this.hass.locale.language)
+                              )
+                              .join(", ")
+                          : this.hass.localize(
+                              "ui.panel.config.cloud.alexa.no_aliases"
+                            )}
```

Comment:
This is an accessibility failure because the button's text doesn't describe what it does.  I think it makes much more sense to display this text alongside a "manage aliases" button.